pyproject.toml

```diff
@@ -42,7 +42,7 @@ dev = [
   "numpydoc",
   "pytest",
   "pytest-cov",
-  "sphinx",
+  "sphinx==0.8.12",
```

Comment:
I plan on reverting this change. I only want to be sure this is causing the breakages on readthedocs.

PS: there are any broken references in the docs!

Reply:
Yep. It is the updated nbsphinx. Fixing it...